packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-stages.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import {
+  Pipeline,
+  Stage,
+  Description,
+  Link,
+  css,
+  cx,
+  spacing,
+  Button,
+  Icon,
+} from '@mongodb-js/compass-components';
+
+import type { RootState } from '../../../modules';
+import { stageAdded } from '../../../modules/pipeline';
+import { changeWorkspace } from '../../../modules/workspace';
+
+const containerStyles = css({
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
+const addStageStyles = css({
+  border: 'none',
+  backgroundColor: 'transparent',
+});
+
+const stagesStyles = css({
+  display: 'flex',
+});
+
+const PipelineStages: React.FunctionComponent<PipelineStagesProps> = ({
+  isEditing,
+  stages,
+  onStageAdded,
+  onChangeWorkspace,
+}) => {
+  if (stages.length === 0) {
+    return (
+      <Description className={containerStyles}>
+        Your pipeline is currently empty. To get started select the
+        <Link
+          className={addStageStyles}
+          as="button"
+          onClick={() => onStageAdded()}
+          hideExternalIcon
+        >
+          first stage.
+        </Link>
+      </Description>
+    );
+  }
```

Comment:
This never shows up because we always have at least one empty stage:

<img width="738" alt="image" src="https://user-images.githubusercontent.com/5036933/158763334-39be7609-8af9-412f-89f9-31bef0abba84.png">